docs/source/examples/assignment_workflows/plot_route_choice.py

```diff
@@ -119,14 +142,53 @@
 # %%
 # We can now perform a computation for single OD pair if we'd like. Here we do one between the first and last centroid
 # as well an an assignment.
-results = rc.execute_single(28, 90, perform_assignment=True)
+results = rc.execute_single(77011, 74089, perform_assignment=True)
 print(results[0])
 
 # %%
 # Because we asked it to also perform an assignment we can access the various results from that
 # The default return is a Pyarrow Table but Pandas is nicer for viewing.
 rc.get_results().to_pandas()
 
+
+# %%
+# let's define a function to plot assignment results
+
+def plot_results(link_loads):
+    import folium
+    import geopandas as gpd
+
+
+    link_loads =link_loads[["link_id", "demand_tot"]]
+    max_load = link_loads["demand_tot"].max()
+    links = gpd.GeoDataFrame(project.network.links.data, crs=4326)
+    links = links.merge(link_loads, on="link_id")
+    links = links[links["demand_tot"] > 0]
+
+    loads_lyr = folium.FeatureGroup("link_loads")
+
+    # Maximum thickness we would like is probably a 10, so let's make sure we don't go over that
+    factor = 10 / max_load
+
+    # Let's create the layers
+    for _, rec in links.iterrows():
+        points = rec.geometry.wkt.replace("LINESTRING ", "").replace("(", "").replace(")", "").split(", ")
+        points = "[[" + "],[".join([p.replace(" ", ", ") for p in points]) + "]]"
+        # we need to take from x/y to lat/long
+        points = [[x[1], x[0]] for x in eval(points)]
+        _ = folium.vector_layers.PolyLine(points, color="red", weight=factor * rec.demand_tot).add_to(loads_lyr)
+
+    long, lat = project.conn.execute("select avg(xmin), avg(ymin) from idx_links_geometry").fetchone()
+
+    map_osm = folium.Map(location=[lat, long], tiles="Cartodb Positron", zoom_start=12)
+    loads_lyr.add_to(map_osm)
+    folium.LayerControl().add_to(map_osm)
+    return map_osm
+
+# %%
+plot_results(rc.get_load_results()[0])
+
```

Comment:
@Jake-Moss , the results for the assignment don't make sense. at all. I am either making a silly mistake, or there is something very wrong indeed.
![image](https://github.com/AequilibraE/aequilibrae/assets/7377666/6dfec421-26ff-48e5-870a-13baaede3c2d)